test/tests/orbit/unit/query-test.js

```diff
@@ -75,5 +75,17 @@ module('Orbit', function() {
         QueryBuilderNotRegisteredException
       );
     });
+
+    test('.from should call toOQE() if available', function(assert) {
```

Comment:
nit pick -- maybe update the test description to align with the new method name.